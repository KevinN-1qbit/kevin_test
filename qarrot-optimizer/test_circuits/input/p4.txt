Rotate -1: ZI
Rotate -1: IZ
Rotate 2: XZ
Rotate -2: IZ
Rotate -2: XI
Rotate 2: IZ
Rotate 2: IX
Rotate 2: IZ
Rotate 2: ZX
Rotate -2: ZI
Rotate -2: IX
Rotate 1: ZI
Rotate 1: IZ
