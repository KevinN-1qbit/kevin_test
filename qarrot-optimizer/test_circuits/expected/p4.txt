Rotate -1: ZI
Rotate -1: IZ
Rotate 1: ZZ
Rotate -1: YY
Measure +: ZZ
Measure -: YY
